src/migration/1610566582191-CreateTables.ts

```diff
@@ -0,0 +1,41 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateTables1610566582191 implements MigrationInterface {
```

Comment:
just for my understanding:  the class name `CreateTables1610566582191` how was that derived?  does the name have any meaning ?

Reply:
I added an npm command to make it a bit easier, so I just executed `npm run typeorm:migration CreateTables`. The name is composed of a descriptive piece (what this migration is doing) + a unix timestamp (when this migration was generated). When the migrations are run, they're run in the order of their timestamps, so it does matter a bit. There's some decently thorough documentation on typeorm's github about how migrations work: https://github.com/typeorm/typeorm/blob/master/docs/migrations.md

Reply:
unix timestamps are hard to read.  is it possible we could use more human readable format?  such as `CreateTables20210113`?

Reply:
edit:  the README description works.

Reply:
Alas, there isn't an option currently to automatically generate it as anything other than a timestamp. The timestamp portion is hardcoded in Typeorm: https://github.com/typeorm/typeorm/blob/master/src/commands/MigrationCreateCommand.ts#L51

Could be worth a PR or raised Issue to add another option.